notification-consumer-services/src/main/java/uk/ac/bbsrc/tgac/miso/notification/consumer/service/mechanism/IlluminaNotificationMessageConsumerMechanism.java

```diff
@@ -167,8 +169,7 @@ public void setAttemptRunPopulation(boolean attemptRunPopulation) {
 
               if (run.has("startDate")) {
                 try {
-                  if (run.get("startDate") != null && !run.getString("startDate").equals("null")
-                      && !"".equals(run.getString("startDate"))) {
+                  if (!"null".equals(run.getString("startDate")) && !isStringEmptyOrNull(run.getString("startDate"))) {
```

Comment:
is it best to check for the actual null first?  That way the shortcircuit `&&` will exit before calling `run.getString("startDate")` and prevent a possible exception.